packages/ui/src/TextField.tsx

```diff
@@ -189,10 +191,24 @@ export function TextField({
     label,
     labelColor,
   };
-  if (type === "date") {
-    value = moment(value).format("MM/DD/YYYY");
-  } else if (type === "height") {
-    value = `${Math.floor(Number(value) / 12)} ft, ${Number(value) % 12} in`;
+  if (value) {
+    if (type === "date") {
+      value = moment(value).format("MM/DD/YYYY");
+    } else if (type === "height") {
+      value = `${Math.floor(Number(value) / 12)} ft, ${Number(value) % 12} in`;
+    } else if (type === "phoneNumber") {
+      // By default, if a value is something like `"(123)"`
+      // then Backspace would only erase the rightmost brace
+      // becoming something like `"(123"`
+      // which would give the same `"123"` value
+      // which would then be formatted back to `"(123)"`
+      // and so a user wouldn't be able to erase the phone number.
+      // This is the workaround for that.
+      const formattedPhoneNumber = new AsYouType("US").input(value);
```

Comment:
The main note on the PR mentioned that this field can support international numbers but it seems we're just focusing on "US" right now (which totally makes sense). Just want to make sure I'm not missing the international support portion. I'm guessing the "AsYouType" library can support it but we're not worried about it right now.

Reply:
Good call out. It is actually just the default value, not locking it to that country. Here are the docs: https://gitlab.com/catamphetamine/libphonenumber-js#class-asyoutypeoptions-or-defaultcountry